config/appConfig.js

```diff
@@ -14,13 +14,15 @@ export default {
     api: process.env.AWS_SQS_API_URL || 'https://sqs.us-east-1.amazonaws.com/224280085904/scsb-item-update-app'
   },
   publicKey:
-		'-----BEGIN PUBLIC KEY-----\n' +
-		'MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAtmIMVmUnbwr/65MYmLGJ\n' +
```

Comment:
This is a different key from that given in https://isso.nypl.org/oauth/key so I updated it